doc/source/plasma-object-store.rst

```diff
@@ -0,0 +1,43 @@
+The Plasma Object Store
+=======================
+
+Plasma is a high-performance shared memory object store originally developed in
+Ray and now being developed in `Apache Arrow`_. See the `relevant
+documentation`_.
+
+Using Plasma with Huge Pages
+----------------------------
+
+On Linux, it is possible to increase the write throughput of the Plasma object
+store by using huge pages. You first need to create a file system and activate
+huge pages as follows.
+
+.. code-block:: shell
+
+  sudo mkdir -p /mnt/hugepages
+  gid=`id -g`
+  uid=`id -u`
+  sudo mount -t hugetlbfs -o uid=$uid -o gid=$gid none /mnt/hugepages
+  sudo bash -c "echo $gid > /proc/sys/vm/hugetlb_shm_group"
+  sudo bash -c "echo 20000 > /proc/sys/vm/nr_hugepages"
+
+You need root access to create the file system, but not for running the object
+store.
+
+You can then start Ray with huge pages on a single machine as follows.
+
+.. code-block:: python
+
+  ray.init(huge_pages_enabled=True, plasma_directory="/mnt/hugepages")
+
+In the cluster case, you can do it as follows.
+
+.. code-block:: shell
+
+  ray start --huge-pages-enabled --plasma-directory=/mnt/hugepages
```

Comment:
this is missing `--head`

Reply:
I omitted it because you also need to do this on non-head nodes.

Reply:
I know, but that command, as is, will run neither on the head nor on the slave nodes. Anybody copy/pasting that command will just see failure:
```
Exception: If --head is not passed in, --redis-address must be provided.
```
With `--head` specified, at least it would work on the head node. Anyway, your updated change is better . Let's just say that we need to pass these command line flags on all nodes where we want to enable hugepage support.